src/keyValueStore.js

```diff
@@ -71,11 +71,16 @@ class KeyValueStore {
         })
       })
     } else {
-      this._db = await this._orbitdb.keyvalue(this._name)
+      this._init()
     }
     return this._db.address.toString()
   }
 
+  async _init() {
+    this._db = await this._orbitdb.keyvalue(this._name)
+    return this._db.address.toString()
```

Comment:
Let's do this._db.load() here ans await the `'ready'` event.

Reply:
👍